release-notes/DxMs/cloudgateway_change_log.md

```diff
@@ -4,9 +4,21 @@ uid: cloudgateway_change_log
 
 # Cloud Gateway change log
 
+#### 4 April 2025 - Enhancement - CloudGateway 2.17.4 - Dependencies updated [ID 42682]
+
+Several dependencies have been updated.
+
+#### 4 April 2025 - Enhancement - CloudGateway 2.17.4 - Improved token refresh [ID 42627]
+
+The tokens to authenticate towards dataminer.services no longer rely on NATS to refresh. This means the tokens will now continue to refresh even when NATS is unavailable.
+
+#### 4 April 2025 - Fix - CloudGateway 2.17.4 - TCP port leak [ID 42458]
+
+A TCP port leak could occur in the CloudGateway i.e. when not being cloud connected.
```

Comment:
What do you mean with "i.e. when not being cloud connected"?

Reply:
When the connection to the cloud (dataminer.services) would not be OK . retries of http requests would more easily increase the amount of tcp ports